gradle/publish-maven.gradle

```diff
@@ -1,105 +1,91 @@
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 if(gradle.ext.shouldSign) {
   apply plugin: 'signing'
-}
-
-install {
-  repositories.mavenInstaller {
-    customizePom(pom, project)
-  }
-}
-
-if(gradle.ext.shouldSign) {
   signing {
+    afterEvaluate { project ->
       useGpgCmd()
-      sign configurations.archives
-  }  
+      sign publishing.publications
+    }
+  }
 }
-
-//upload to sonatype configuration
-uploadArchives {
+//define the repositories on global level, not per project, or you get many of them
+publishing {
   repositories {
-    mavenDeployer {
-      if(gradle.ext.shouldSign) {
-        beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-      }
+    maven {
+      name = "ossrh"
+      //  if(gradle.ext.shouldSign) {
+      //    beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+      //  }
```

Comment:
This is commented for a reason?

Reply:
Yes, it's no longer necessary. I will remove this.